app/Platform/Data/GameData.php

```diff
@@ -28,7 +37,15 @@ public static function fromGame(Game $game): self
             title: $game->title,
             badgeUrl: Lazy::create(fn () => $game->badge_url),
             forumTopicId: Lazy::create(fn () => $game->ForumTopicID),
-            system: Lazy::create(fn () => SystemData::fromSystem($game->system))
+            system: Lazy::create(fn () => SystemData::fromSystem($game->system)),
+            achievementsPublished: Lazy::create(fn () => $game->achievements_published),
+            pointsTotal: Lazy::create(fn () => $game->points_total),
+            pointsWeighted: Lazy::create(fn () => $game->TotalTruePoints),
+            releasedAt: Lazy::create(fn () => $game->released_at),
+            releasedAtGranularity: Lazy::create(fn () => $game->released_at_granularity),
+            lastUpdated: Lazy::create(fn () => $game->lastUpdated),
+            numVisibleLeaderboards: Lazy::create(fn () => $game->num_visible_leaderboards ?? 0),
+            numUnresolvedTickets: Lazy::create(fn () => $game->num_unresolved_tickets ?? 0),
```

Comment:
This ties into another comment I had: Why are all of these lazy if the data is simple data types that are readily available? The only one that looks potentially complex is `system`, but I'm pretty sure that data is readily available at this point too.

Reply:
Tried to do a bit of explanation in my comment here https://github.com/RetroAchievements/RAWeb/pull/2686#discussion_r1757679671 but want to add another follow-up on this comment.

A logical question might be: "why use these `Data` classes at all?" The primary benefit we get from them is end-to-end type safety, something we definitely do not have in our Blade UIs right now (see: mistyped routes causing pages to crash, us needing to constantly comment on prop types, etc)

You can prove this out very easily. Open _GameData.php_ and _generated.d.ts_. Remove any of those `Lazy|` from any of the fields and save the file. In the terminal, run `composer types`. _generated.d.ts_ will update with the new correct type for the UI to use, and if the UI is mistyped, `tsc` will fail.